src/build.py

```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
```

Comment:
Yeah, except it's not really a prefix, it's more of an infix (it is actually used as a subdir of the `INSTALL_DIR` which is what you might call the global prefix).